packages/server/prisma/schema.prisma

```diff
@@ -12,15 +12,18 @@ datasource db {
 }
 
 model User {
-  id          String    @default(uuid()) @id
+  id          String @unique @default(uuid())
   project_id  String
-  username    String?
-  email       String?
+  username    String? @unique
+  email       String? @unique
```

Comment:
Maybe remove the @unique attribute. Not only the same username may be used at different project as Collin mentioned, but it will also trigger problems when username/email field is not required. When there are multiple records that do not have username or email, it is not unique anymore.

Reply:
Fixed